extensions/content-health-monitor/content-health-monitor.qmd

```diff
@@ -427,9 +443,9 @@ def create_report_display(result_data, check_time_value):
 
 # Create the about content text
 about_content = """<div>
-This report uses the publisher's API key to monitor an application GUID, verifies that the monitored app is 
-reachable at its content URL, and reports the results. If run on a schedule, reports will be emailed when the monitored 
-application reports a failure.
+This report uses the publisher’s API key to monitor a single piece of content. It checks whether the content is 
+reachable, but does not validate its functionality. If scheduled to run regularly, it will send an email alert if the 
```

Comment:
Lightest ever suggestion but this makes the sentence less "if" heavy :)

```suggestion
reachable, but does not validate its functionality. When scheduled to run regularly, it will send an email alert if the 
```